build-config/make/u-boot.make

```diff
@@ -139,7 +139,7 @@ $(UBOOT_BUILD_STAMP): $(UBOOT_IMAGE)
 u-boot-install: $(UBOOT_INSTALL_STAMP)
 $(UBOOT_INSTALL_STAMP): $(UBOOT_BUILD_STAMP)
 	$(Q) echo "==== Installing u-boot ($(MACHINE_PREFIX)) ===="
-	$(Q) cp -v $(UBOOT_IMAGE) $(UBOOT_INSTALL_IMAGE)
+	$(Q) cp -v $(UBOOT_BUILD_DIR)/$(UBOOT_MACHINE)/u-boot-dtb.bin $(UBOOT_INSTALL_IMAGE)
```

Comment:
I want to add a variable, if it is u-boot-dtb.bin, then assign the variable to u-boot-dtb.bin, else assign it to u-boot.bin. do you have some suggestion? and any suggestion about where this variable add?